src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +328,70 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const renderCustomBadge = () => {
+        const badgeButton = (
+          // @ts-ignore - EuiBadge href type issue
+          <EuiBadge
+            {...getManagedContentBadge(dashboardManagedBadge.getText(), false)}
+            onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+          >
+            {managedText}
+          </EuiBadge>
+        );
+        return (
+          <EuiPopover
+            button={badgeButton}
+            isOpen={isPopoverOpen}
+            closePopover={() => setIsPopoverOpen(false)}
+            className="eui-hideFor--s eui-hideFor--xs"
+            data-test-subj="managedContentPopover"
+          >
+            <EuiFlexItem>
+              <EuiText
+                size="s"
+                aria-label={text}
+                css={css`
+                  max-width: 300px;
+                `}
+              >
+                {text}
+              </EuiText>
```

Comment:
I don't think the `max-width` styling belongs on the `EuiText` component - can we not apply this via `PopoverPanelProps` (see [the docs](https://eui.elastic.co/#/layout/popover))?